t_wada.js

```diff
@@ -79,7 +79,16 @@ function main() {
     canvas.height = canvasSize.height;
     var ctx = canvas.getContext('2d');
     draw(ctx, canvasSize, textarea.value);
+    canvas.addEventListener("click", function(event) {
+        this.toBlob(function(blob) {
```

Comment:
I think that `event.currentTarget` is better.
Maybe, We can use arrow function instead of `function` expression in the future.